app/components/buttons/secondary_component.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Buttons
+  class SecondaryComponent < ButtonComponent
```

Comment:
What do you think about using custom constructors rather than custom classes? I.e. `ButtonComponent.new_secondary_button` or `Component#remove_button`

Reply:
I have to think about it.

In order to be used as slots it is more convenient for them to be class names, but they could be used in a slot with a custom factory method as well, called in a block.

🤔